quantum/process_keycode/process_unicode_common.c

```diff
@@ -118,6 +118,9 @@ void unicode_input_finish(void) {
   case UC_WIN:
     unregister_code(KC_LALT);
     break;
+  case UC_WINC:
+    tap_code(KC_SPC);
+    break;
```

Comment:
Instead of adding new code, I suggest just putting `case UC_WINC:` below `case UC_LNX:` on line 115 since the code is the same. In other words:

```c
case UC_LNX:
case UC_WINC:
  tap_code(KC_SPC);
  break;
```

Reply:
yes, this is better. Thank you